WTWSMS/history/provinces/666 - Gilan.txt

```diff
@@ -16,8 +16,8 @@ b_masouleh = castle
 #b_rudbar = castle
 
 #Misc
-culture = aramean
-religion = manichean
+culture = kurdish
+religion = yazidi
```

Comment:
Listing Yazidis as a Muslim heresy in this Mod is ahistorical. Yazidis existed as majorities in this time period.